Messege box

GoralTomaszGorski opened the pull request:
message from prepared typos to fix,
controller is ready 
but still problem with MessageService tomorrow I will work on it

gorskimarcin96 committed:
massage-box function preparing

gorskimarcin96 committed:
massage-box function preparing

gorskimarcin96 committed:
massage-box function preparing

GoralTomaszGorski committed:
massage-box function preparing

GoralTomaszGorski committed:
massage-box function preparing

GoralTomaszGorski committed:
massage-box function preparing

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
addMeetingWithNewPatient function fixed.
massage-box function preparing.
templates style improved.

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

massage form, controller added, MessageService to fix

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

massage form, controller added, MessageService to fix

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

massage form, controller added, MessageService to fix

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

massage form, controller added, MessageService to fix

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

# Conflicts:
#	src/main/java/goral/psychotherapistoffice/domain/messeges/MessageService.java
#	src/main/java/goral/psychotherapistoffice/domain/messeges/dto/MessageDto.java
#	src/main/java/goral/psychotherapistoffice/web/user/MessageController.java
#	src/main/resources/templates/user/mail-form.html

GoralTomaszGorski commented:
[aea1ed3](https://github.com/GoralTomaszGorski/psychotherapist_office/pull/21/commits/aea1ed3ef785bfe01554aa04bdfb7ed7c28f92c3)
to repair tommorow

GoralTomaszGorski committed:
email sender working

tomorrow some methods I will split into another class to code clean

GoralTomaszGorski committed:
email sender working

tomorrow some methods I will split into another class to code clean

GoralTomaszGorski commented:
however, today I made this box

fuc***g token from gmail was necessary ;)

gorskimarcin96 commented on src/main/java/goral/psychotherapistoffice/domain/messeges/MessageService.java:
```diff
@@ -0,0 +1,56 @@
+        properties.put("mail.smtp.port", "587");
+        properties.put("mail.smtp.host", "smtp.gmail.com");
+
+        final String username = "tomasz.gorski88@gmail.com";
```
in php usually we are creating hide .env file and we can move this values to file

https://www.geeksforgeeks.org/environment-variables-in-java/

GoralTomaszGorski committed:
email sender working
the methods have been split into another class to clean up the code
password and email in other class

GoralTomaszGorski committed:
email sender working
the methods have been split into another class to clean up the code
password and email in other class

GoralTomaszGorski commented:
the methods have been split into another class to clean up the code with token and email

GoralTomaszGorski committed:
Merge remote-tracking branch 'origin/messege-boc' into messege-box

# Conflicts:
#	src/main/java/goral/psychotherapistoffice/web/user/MessageController.java
#	src/main/resources/templates/user/mail-form.html
